runtimes/manager.py

```
@@ -48,39 +48,18 @@ def unregister(self, runtime_id: str) -> None:
-    def get_decision_log(self, limit: int = 100) -> list[dict[str, Any]]: return self._router.get_decision_log(limit)
-    def health_summary(self) -> list[dict[str, Any]]: return self._health.all_health()
-    async def verify_all(self) -> list[dict[str, Any]]: return await self._health.verify_all()
-    async def refresh_runtime_health(self, runtime_id: str) -> dict[str, Any] | None:
-        adapter = self._registry.get(runtime_id)
-        if adapter is None: return None
+    def select_runtime(self, task_type: str, preferred_id: str | None = None) -> tuple[RuntimeAdapter | None, list[dict]]:
+        return self._router._pick_runtime(task_type, preferred_id)
+
+    async def get_runtime_health(self, runtime_id: str) -> dict | None:
         circuit = self._health._circuits.get(runtime_id)
         if circuit: circuit.record_success()
```

Ruff (0.15.13)

[error] Multiple statements on one line (colon)

(E701)